client/sites/components/sites-dashboard.tsx

```diff
@@ -276,68 +278,73 @@ const SitesDashboard = ( {
 
 	// Perform sorting actions
 	const sortedSites = useSitesListSorting( currentStatusGroup, {
-		sortKey: siteSortingKeys.find( ( key ) => key.dataView === dataViewsState.sort?.field )
+		sortKey: siteSortingKeys.find( ( key ) => key.dataView === viewState.sort?.field )
 			?.sortKey as SitesSortKey,
-		sortOrder: dataViewsState.sort?.direction || undefined,
+		sortOrder: viewState.sort?.direction || undefined,
 	} );
 
 	// Filter sites list by search query.
 	const filteredSites = useSitesListFiltering( sortedSites, {
-		search: dataViewsState.search,
+		search: viewState.search,
 	} );
 
 	const paginatedSites =
-		dataViewsState.page && dataViewsState.perPage
+		viewState.page && viewState.perPage
 			? filteredSites.slice(
-					( dataViewsState.page - 1 ) * dataViewsState.perPage,
-					dataViewsState.page * dataViewsState.perPage
+					( viewState.page - 1 ) * viewState.perPage,
+					viewState.page * viewState.perPage
 			  )
 			: filteredSites;
 
 	const onboardingTours = useOnboardingTours();
 
-	useInitializeDataViewsPage( dataViewsState, setDataViewsState );
+	useInitializeDataViewsPage( viewState, setViewState );
 
 	// Update URL with view control params on change.
 	useEffect( () => {
 		const queryParams = {
-			search: dataViewsState.search?.trim(),
+			search: viewState.search?.trim(),
 			status: statusSlug,
-			page: dataViewsState.page && dataViewsState.page > 1 ? dataViewsState.page : undefined,
-			'per-page': dataViewsState.perPage === DEFAULT_PER_PAGE ? undefined : dataViewsState.perPage,
+			page: viewState.page && viewState.page > 1 ? viewState.page : undefined,
+			'per-page': viewState.perPage === DEFAULT_PER_PAGE ? undefined : viewState.perPage,
 		};
 
 		window.setTimeout( () => handleQueryParamChange( queryParams ) );
-	}, [ dataViewsState.search, dataViewsState.page, dataViewsState.perPage, statusSlug ] );
+	}, [ viewState.search, viewState.page, viewState.perPage, statusSlug ] );
 
 	// Update site sorting preference on change
 	useEffect( () => {
-		if ( dataViewsState.sort?.field ) {
+		if ( viewState.sort?.field ) {
 			onSitesSortingChange( {
-				sortKey: siteSortingKeys.find( ( key ) => key.dataView === dataViewsState.sort?.field )
+				sortKey: siteSortingKeys.find( ( key ) => key.dataView === viewState.sort?.field )
 					?.sortKey as SitesSortKey,
-				sortOrder: dataViewsState.sort.direction || 'asc',
+				sortOrder: viewState.sort.direction || 'asc',
 			} );
 		}
-	}, [ dataViewsState.sort, onSitesSortingChange ] );
+	}, [ viewState.sort, onSitesSortingChange ] );
 
 	// Manage the closing of the preview pane
 	const closeSitePreviewPane = useCallback( () => {
-		if ( dataViewsState.selectedItem ) {
-			setDataViewsState( { ...dataViewsState, type: DATAVIEWS_TABLE, selectedItem: undefined } );
+		if ( selectedSite ) {
+			// setViewState( { ...viewState, type: 'table' } );
+			pagejs( '/sites' );
```

Comment:
This also clears the query params, which shouldn't.

Reply:
Should be able to pass these through.

Reply:
a1c366593118b753865c200eb0589b2bb0f7a518